### ISSUE TITLE
Add Ethereum Contract ABI Converter

### ISSUE THREAD

**Issue by davyvanroy** [Cindercloud/Cindercloud#59]
When verifying your contract on etherscan you need to add the constructor abi parameters.
It would be nice to add this to the tools section

to get an idea: https://github.com/thanhson1085/convert-contract-abi-online

**Comment by Qkyrie** [comment]
Good call, will be quite handy once we implement contract verification ourselves
